plutus-core/executables/Common.hs

```diff
@@ -363,17 +373,23 @@ writeProgram ::
    Flat (a ()),
    PP.PrettyBy PP.PrettyConfigPlc (a b)) =>
    Output -> Format -> PrintMode -> a b -> IO ()
-writeProgram outp Textual mode prog      = writeToFileOrStd outp mode prog
+writeProgram outp Textual mode prog      = writePrettyToFileOrStd outp mode prog
 writeProgram outp (Flat flatMode) _ prog = writeFlat outp flatMode prog
 
-writeToFileOrStd ::
+writePrettyToFileOrStd ::
   (PP.PrettyBy PP.PrettyConfigPlc (a b)) => Output -> PrintMode -> a b -> IO ()
-writeToFileOrStd outp mode prog = do
+writePrettyToFileOrStd outp mode prog = do
   let printMethod = getPrintMethod mode
   case outp of
         FileOutput file -> writeFile file . Prelude.show . printMethod $ prog
         StdOutput       -> print . printMethod $ prog
 
+writeToFileOrStd ::
```

Comment:
This is currently still a bit wrong, since it overwrites any previous writes. I want to rewrite this to pass around a handle instead.